evap/staff/templates/staff_evaluation_person_management.html

```diff
@@ -91,7 +91,7 @@ <h6 class="card-subtitle mb-2 text-muted">{% trans 'From other evaluation' %}</h
                         {% include 'bootstrap_form.html' with form=contributor_copy_form wide=True %}
                     </div>
                     <div class="card-footer text-center">
-                        <button name="operation" value="copy-contributors" type="submit" class="btn btn-sm btn-primary form-submit-btn">{% trans 'Copy contributors' %}</button>
+                        <button name="operation" value="copy-contributors" type="button" onclick="copyContributorsModalShow('copy-contributors', '{{ field }}');" class="btn btn-sm btn-primary form-submit-btn">{% trans 'Copy contributors' %}</button>
```

Comment:
What is `{{ field }}` here? As far as I can see, this should be undefined. In the other copy modal above, you pass no second argument.